index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Portfolio website</title>
+    <link rel="stylesheet" href="portfolio-main.css">
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <div class="status-bar ">
+        <p>9:41</p>
+        <img class="cellular" src="images/cellular.png" alt="cellular Connection icon"/>
+        <img class="wifi" src="images/Wifi.png" alt="Wifi icon"/>
+        <img class="battery" src="images/battery.png" alt="Battery icon"/>
+      </div>
+      <div class="toolbar">
+        <p>Chere Lema</p>
+        <img src="images/normal_button.png" alt="Normal Button"/>
+      </div>
+    </nav>
+
+    <header class="headline">
+      <section >
+        <h1 class="primary-text">Hey There. I&rsquo;m Chere</h1>
+        <h2 class="support-text">I&rsquo;am a Software Developer</h2>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+      </section>
+      <section class="social-icons">
+        <img src="images/github.png" alt="Github Icon"/>
+        <img src="images/Linkedin.png" alt="Linkedin Icon"/>
+        <img src="images/angel_list.png" alt="Angel list Icon"/>
+        <img src="images/twitter.png" alt="Twitter Icon"/>
+        <img src="images/medium.png" alt="Medium Icon"/> 
+      </section>
+
+      <section class="illustration">
+        <img src="images/illustration.png" alt="illustration"/>
+      </section>     
+    </header>
```

Comment:
- [ ] I think the `<header>` tag should be used just for the header and not for the content, 🎲  instead of it you could use the `<section>` tag, and make sure within each `<section>` tag should be a `<hx>` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. ⚡